vite.config.js

```diff
@@ -1,11 +1,17 @@
 import { defineConfig } from 'vite';
 import laravel from 'laravel-vite-plugin';
+import path from 'path'; // node
 
 export default defineConfig({
     plugins: [
         laravel({
-            input: ['resources/css/app.css', 'resources/js/app.js'],
+            input: ['resources/css/app.scss', 'resources/js/app.js'],
```

Comment:
Please fix the vite config... The file 'resources/css/app.scss' does not exists.. I see from the list 'resources/css/app.css' and 'resources/sass/app.scss' (use one of those 2 ones..).